src/components/Book.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+import './Styles/Book.css';
+import BookProgress from './BookProgress';
+
+const Book = (props) => {
+  const {
+    title,
+    author,
+    id,
+    category,
+  } = props;
+  const dispatch = useDispatch();
+
+  const remove = () => {
+    dispatch(removeBook(id));
+  };
+
+  return (
+    <div className="book-wrapper">
+      <div className="book-detail">
+        <h4 className="book-genre">{category}</h4>
+        <h3 className="book-title">{title}</h3>
+        <p className="book-author">{author}</p>
+        <div className="book-controls">
+          <button type="button" className="button-outline button-comment">
+            Comments
+          </button>
+          <div className="vertical-separator" />
+          <button type="button" className="button-outline button-remove" onClick={remove}>
+            Remove
+          </button>
+          <div className="vertical-separator" />
+          <button type="button" className="button-outline button-edit">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="book-progress">
+        <BookProgress />
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ ] Kindly improve your styling to match the design on zeplin. Be sure to follow the font styling, and spacing between the elements, and colors exactly as we have on the design.

  Your Design| Zeplin Design
  :---:|:---:
  ![urwp](https://user-images.githubusercontent.com/54848363/216533015-6865e859-d784-4caf-a5b8-bd635d4c409f.jpg) | ![etq](https://user-images.githubusercontent.com/54848363/216533008-7628e7bc-7817-4f66-a76e-473a86bba93a.jpg)

Reply:
Done. Pls Approve! Thanks.